src/seedsigner/views/scan_views.py

```diff
@@ -12,19 +13,24 @@
 
 
 class ScanView(View):
-    def run(self):
-        from seedsigner.gui.screens.scan_screens import ScanScreen
+    def __init__(self):
+        super().__init__()
+
+        # Set up the QR decoder here so we can inject data into it in the test suite's
+        # `before_run`.
+        self.wordlist_language_code = self.settings.get_value(SettingsConstants.SETTING__WORDLIST_LANGUAGE)
+        self.decoder = DecodeQR(wordlist_language_code=self.wordlist_language_code)
```

Comment:
This is a flowtest-todo pattern that I'm just noticing for the first time.  Perhaps include this in an eventual flow-test writeup.